pkg/db/db_releases.go

```diff
@@ -0,0 +1,681 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package db
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/freiheit-com/kuberpult/pkg/logger"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"slices"
+	"strings"
+	"time"
+)
+
+type DBReleaseMetaData struct {
+	SourceAuthor    string
+	SourceCommitId  string
+	SourceMessage   string
+	DisplayVersion  string
+	UndeployVersion bool
+	IsMinor         bool
+	CiLink          string
+	IsPrepublish    bool
+}
+
+type DBReleaseManifests struct {
+	Manifests map[string]string
+}
+
+type DBReleaseWithMetaData struct {
+	ReleaseNumber uint64
+	Created       time.Time
+	App           string
+	Manifests     DBReleaseManifests
+	Metadata      DBReleaseMetaData
+	Environments  []string
+}
+
+// SELECTS
+
+func (h *DBHandler) DBSelectAnyRelease(ctx context.Context, tx *sql.Tx, ignorePrepublishes bool) (*DBReleaseWithMetaData, error) {
+	selectQuery := h.AdaptQuery(`
+	SELECT created, appName, metadata, releaseVersion, environments 
+	FROM releases 
+	LIMIT 1;`)
```

Comment:
```suggestion
		SELECT created, appName, metadata, releaseVersion, environments 
		FROM releases 
		LIMIT 1;`)
```

please separate the query visually from the rest of the code.

Reply:
(same for other queries)

Reply:
Done